src/main/java/org/zalando/nakadi/controller/SubscriptionController.java

```diff
@@ -110,17 +115,32 @@ public SubscriptionController(final SubscriptionDbRepository subscriptionReposit
     @RequestMapping(method = RequestMethod.GET)
     public ResponseEntity<?> listSubscriptions(
             @Nullable @RequestParam(value = "owning_application", required = false) final String owningApplication,
+            @Nullable @RequestParam(value = "event_type", required = false) final Set<String> eventTypes,
+            @RequestParam(value = "limit", required = false, defaultValue = "20") final int limit,
+            @RequestParam(value = "offset", required = false, defaultValue = "0") final int offset,
             final NativeWebRequest request) {
 
         if (!featureToggleService.isFeatureEnabled(HIGH_LEVEL_API)) {
```

Comment:
I think we should keep the toggle till we finish the HILA and will be ready to show that to our users on live.